pkg/clickhouse/source.go

```diff
@@ -102,14 +102,56 @@ func toStringSlice(iSlice []interface{}) []string {
 	return values
 }
 
-func (s Source) WriteChunk(_ string, _ io.Reader) error {
-	// TODO
-	return errors.New("not implemented")
+func (s Source) WriteChunk(_ string, r io.Reader) error {
+	ct, err := s.ColumnTypes()
```

Comment:
I think we can get column types once at instantiation instead of for each chunk. Fewer requests = less load